src/actions/ws_movement.ts

```diff
@@ -13,6 +13,11 @@ const createSerializedKey = ShortcutRegistry.registry.createSerializedKey.bind(
   ShortcutRegistry.registry,
 );
 
+/**
+ * The distance to move the cursor when the cursor is on the workspace.
+ */
+const WS_MOVE_DISTANCE = 40;
```

Comment:
Moved from the class, but not sure if this was intended to be mutable as a sort of configuration.